docs/todos.md

```diff
@@ -31,3 +31,76 @@ If an error is fixed manually, `ember-template-lint` will let you know that ther
 ```bash
 ember-template-lint . --fix
 ```
+
+### Assigning Due Dates
+
+TODOs can be created with optional due dates. These due dates allow for TODOs to, over a period of time, 'decay' the severity to a **warning** and/or **error** after a certain date. This helps ensure that TODOs are created but not forgotten, and can allow for better managing incremental roll-outs of large-scale or slow-to-fix rules.
+
+Due dates can be configured in multiple ways, but all specify integers for `warn` and `error` to signify the number of days from the TODO created date to decay the severity.
+
+:bulb: Both `warn` and `error` are optional. The value for `warn` should be greater than the value of `error`.
+
+1. Via package.json configuration
+
+```json
+{
+  "lintTodo": {
+    "decayDays": {
+      "warn": 5,
+      "error": 10
+    }
+  }
+}
+```
+
+2. Via environment variables
+
+```bash
+TODO_DAYS_TO_WARN="5" TODO_DAYS_TO_ERROR="10" ember-template-lint . --update-todo
+```
+
+3. Via command line options
+
+```bash
+ember-template-lint . --update-todo --todo-days-to-warn=5 --todo-days-to-error=10
+```
```

Comment:
I think _in general_ we don't expect ember-template-lint users to leverage these environment variables, think it would be ok to flip 2 and 3 (so we mention the command line flags first)? You already explain the precedence just below, so it _shouldn't_ make it more confusing to folks...

What do you think?

Reply:
I agree. I think you and I should decide how we want to publicize this functionality. Let's do that in a follow up.